modify grammar

zr-182 opened the pull request.

zr-182 commented:
您发给我的邮件已经收到。This is an automatic reply, confirming that your e-mail was received.Thank you

zr-182 commented:
I have read the CLA Document and I hereby sign the CLA

zr-182 commented:
I have read the CLA Document and I hereby sign the CLA

zr-182 commented:
recheck

zr-182 commented:
> 

recheck

zr-182 commented:
recheck

zr-182 commented:
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/secretflow/.github/blob/main/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> _**I have read the CLA Document and I hereby sign the CLA**_
> 
> You can retrigger this bot by commenting **recheck** in this Pull Request

recheck

zr-182 reviewed (commented):
recheck

6fj commented on README.md:
```diff
@@ -2,7 +2,7 @@
 [![CircleCI](https://dl.circleci.com/status-badge/img/gh/secretflow/yacl/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/secretflow/yacl/tree/main)
 
-YACL (Yet Another Crypto library) is a C++ library that contains modules and utilities which other SecretFlow code depends on.
+YACL (Yet Another Crypto library) is a C++ library that contains modules and utilities which other SecretFlow codes depend on.
```
I don't think "codes" is correct here.

e.g. we usually say "other peoples' code" instead of "other peoples' codes".

6fj commented on REPO_LAYOUT.md:
```diff
@@ -8,7 +8,7 @@ YACL is a fundamental library used in the SecretFlow stack. The major components
     - [dpf/](yacl/crypto/primitives/dpf/): Distributed Point Function (DPF).
     - [ot/](yacl/crypto/primitives/ot/): Oblivious transfer and oblivious transfer extensions.
-  - [tools/](yacl/crypto/tools/): Theoretical cryptographic tools such as Random Oracle (RO), Pseudorandom Generator (PRG).
+  - [tools/](yacl/crypto/tools/): Theoretical cryptographic tools such as Random Oracle (RO), Pseudo-Random Generator (PRG).
```
Pseudorandom Generator is correct.

6fj commented on yacl/base/int128.h:
```diff
@@ -34,8 +34,8 @@ namespace details {
   // Casting an unsigned integer to a signed integer of the same
-  // width is implementation defined behavior if the source value would not fit
-  // in the destination type. We step around it with a roundtrip bitwise not
+  // width is an implementation defined behavior if the source value would not fit
```
s/implementation defined/implementation-defined

6fj commented on yacl/crypto/base/hash/hash_interface.h:
```diff
@@ -40,7 +40,7 @@ enum class HashAlgorithm : int {
 // Data may be added to an instance of HashInterface via the Update() method at
 // any point during the object's lifetime. A user may call the CumulativeHash()
-// method to get a hash of all data added to the object since its creation or
+// method to get a hash of all data added to the object from its creation or
```
since is good to go.

6fj commented:
recheck

6fj commented:
Hi @zr-182 ,

I find the changes are committed by @IIIImp . Please double-check your git settings.
Thanks.

zr-182 commented:
Hi @6fj ,

I added new commits under your advice . Please check if they r OK . 

And , as you said , the first commit is from @IIIImp . I forgot to change the git account but now I have changed it. I wonder if it is all right.

IIIImp commented:
I have read the CLA Document and I hereby sign the CLA

zr-182 commented:
recheck
